panel/models/terminal.ts

```diff
@@ -0,0 +1,74 @@
+// See https://docs.bokeh.org/en/latest/docs/reference/models/layouts.html
+import { HTMLBox, HTMLBoxView } from "@bokehjs/models/layouts/html_box"
+
+// See https://docs.bokeh.org/en/latest/docs/reference/core/properties.html
+import * as p from "@bokehjs/core/properties"
+import { div } from "@bokehjs/core/dom";
+
+
+// The view of the Bokeh extension/ HTML element
+// Here you can define how to render the model as well as react to model changes or View events.
+export class TerminalView extends HTMLBoxView {
+    model: Terminal
+    term: any // Element
```

Comment:
Not sure about your build/module system but you might be able to do this to bring the type in without the implementation:

```ts
import type { Terminal } from 'xterm';
```

Reply:
Thanks. I will try it out.